armi/reactor/tests/test_components.py

```diff
@@ -1122,6 +1247,12 @@ def test_dimensionThermallyExpands(self):
 
 
 class TestHoledSquare(TestHoledRectangle):
+    """Test holed square shaped component.
+
+    .. test:: Holed hexagon shaped component
+        :id: T_ARMI_COMP_SHAPES6
+        :tests: R_ARMI_COMP_SHAPES
+    """
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.